scripts/o3de/o3de/repo_properties.py

```diff
@@ -293,6 +293,44 @@ def _auto_update_json(object_type: str or list,
             _edit_objects(object_name, validation.valid_o3de_template_json, repo_json, expected_files.get("template.json"))
     return 0
 
+def print_repo_diff(repo_json: dict,
+             repo_json_original: dict):
+    
+    if repo_json_original == repo_json:
+            logger.warning('No difference is found between your current repo.json file and your dry run')
+            return 0
```

Comment:
I would not consider not having any differences to be an error or warning. I think its valid to just not print anything.